services/requests/request_definitions/parsers.go

```diff
@@ -0,0 +1,167 @@
+package request_definitions
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/leanspace/terraform-provider-leanspace/helper"
+)
+
+func (feasibilityConstraintDefinition *RequestDefinition) ToMap() map[string]any {
+	feasibilityConstraintDefinitionMap := make(map[string]any)
+	feasibilityConstraintDefinitionMap["id"] = feasibilityConstraintDefinition.ID
+	feasibilityConstraintDefinitionMap["name"] = feasibilityConstraintDefinition.Name
+	feasibilityConstraintDefinitionMap["description"] = feasibilityConstraintDefinition.Description
+	feasibilityConstraintDefinitionMap["cloned"] = feasibilityConstraintDefinition.FeasibilityConstraintDefinitions
```

Comment:
I'm not sure cloned is an actual thing but in case it is:
```suggestion
	feasibilityConstraintDefinitionMap["cloned"] = feasibilityConstraintDefinition.Cloned
```

Reply:
removed